create.go

```diff
@@ -92,6 +93,19 @@ func create(containerID, bundlePath, console, pidFilePath string) error {
 		return err
 	}
 
+	// config.json provides a cgroups path that has to be used to create "tasks"
+	// and "cgroups.procs" files. Those files have to be filled with a PID, which
+	// is shim's in our case. This is mandatory to make sure there is no one
+	// else (like Docker) trying to create those files on our behalf. We want to
+	// know those files location so that we can remove them when delete is called.
+	if ociSpec.Linux.CgroupsPath == "" {
+		return fmt.Errorf("CgroupsPath should not be empty")
```

Comment:
Are we sure `CgroupsPath` is an OCI MUST ? Surely Docker uses it, but I don't see where it's stated as a mandatory setting in OCI.
@jodh-intel Looking for your OCI expert input here...